### ISSUE TITLE
Samples, documentation

### ISSUE THREAD

**Issue by xperiandri** [fsprojects/FSharp.ViewModule#21]
Are there any samples of how to use this?
API docs would be helpful too.

**Comment by ReedCopsey** [comment]
@xperiandri Yes, I agree - there's a woeful lack of documentation.  To be honest, I haven't been incredibly motivated to write them as I've been focused more on Gjallarhorn (since I think it's a better model for data binding scenarios).

There are quite a few samples, however, here: https://github.com/fsprojects/FsXaml/tree/master/demos  The three "Wpf..." demos all use FSharp.ViewModule.
